controllers/openshift_controller.go

```diff
@@ -1728,6 +1728,39 @@ func (r *KataConfigOpenShiftReconciler) clearFailedStatus(status kataconfigurati
 	return status
 }
 
+func (r *KataConfigOpenShiftReconciler) createAuthJsonSecret() error {
+	var err error = nil
+
+	pullSecret := &corev1.Secret{}
+	err = r.Client.Get(context.TODO(), types.NamespacedName{Name: "pull-secret", Namespace: "openshift-config"}, pullSecret)
+	if err != nil {
+		r.Log.Error(err, "Error fetching pull-secret")
```

Comment:
Based on suggestions in other PRs from @gkurz and @pmores, can you change the `r.Log.Error` usage to `r.Log.Info`. 
Rest looks good to me.